### ISSUE TITLE
Annotations stem styling syntax is different from docs.

### ISSUE THREAD

**Issue by moustacheful** [google/google-visualization-issues#2038]
Hey!
Just a heads up.
Currently on every part regarding annotations there's mention of the possibility of doing this to globally style annotations' stems: 

``` js
options = {
    annotations:{
        stem:{
            color: 'red',
            length: 20
        }
    }
}
```

Apparently, this is not the case, and the correct syntax is (currently) the following

``` js
options = {
    annotations:{
        stemColor: 'red',
        stemLength: 20
    }
}
```

I'm having a similar issue with the `highContrast` option, which doesn't seem to work.

**Comment by dlaliberte** [comment]
I forget when we made the change, but the stem: { ... } form might only apply to version 42.  If you load 1 or 1.1 now, you will get v42.  Later today, 1.0 will also be v42.

**Comment by moustacheful** [comment]
Ah, I see. Have you considered versioned documentation (similar to the angular documentation https://docs.angularjs.org/api), now that you've implemented the version freezing feature?
I would love to see this!

**Comment by dlaliberte** [comment]
We really do need to support versioned documentation, now that frozen versions are becoming available. Not sure when we will be able to get to it.

**Comment by nbering** [comment]
I agree.  I've already provided support for one or two users where to issue turned out to be that the documentation was showing something that depended on `1.1`, and they were using `1`.

**Comment by peterlitvak** [comment]
This seems to be pretty old issue but I have a problem with ``highContrast`` which doesn't seem to work, what would be the correct property name for it?
